test/corecel/math/IllinoisRootFinder.test.cc

```diff
@@ -0,0 +1,96 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020-2024 UT-Battelle, LLC, and other Celeritas developers.
```

Comment:
Since all these files are new, you can just mark them as 2024 :) They're normally generated automatically with `scripts/dev/celeritas-gen.py`.